README.md

```diff
@@ -13,7 +13,7 @@ Looking for help? Find resources [from the community](https://graphql.org/commun
 
 ## Getting Started
 
-An overview of GraphQL in general is available in the
+An overview of GraphQL, in general, is available in the
```

Comment:
issue: This changes the meaning of the original sentence.

"an overview of GraphQL in general" is equivalent to "a general overview of GraphQL". By adding the commas, we are changing the meaning to "an overview of GraphQL, in most situations".

The original sentence is syntactically valid, although changing it to "A general overview of GraphQL" would be more succinct and clearer.

Reply:
Agreed. Updated!